credenciales.py

```diff
@@ -0,0 +1,15 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+class Credenciales :
+    USUARIO:str = os.getenv("USUARIO")
+    PASSWORD:str = os.getenv("PASSWORD")
+    HOST:str = os.getenv("HOST")
+    POERT:str = os.getenv ("POERT")
```

Comment:
```suggestion
    POERT:str = os.getenv ("PORT")
```

Debería ser `"PORT"`.